Alameen688/mydiary-spa
# feat(login): user should be able to login

#### What does this PR do?
- Implements login functionality
#### Description of Task to be completed?
- An authenticated user should not be able to access sign up and login page
- Allow users with valid credentials to login and be redirected to the entries page on successful login.
#### How should this be manually tested?
- Pull this branch
- Run `npm run dev`
- Go to `localhost:8080/login` enter a registered account information
- You should be taken to `/entries` on successful login
- If there was an error on login attempt, the error should be displayed on the page and you should be prevented from logging in.
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?

[ #162190563](https://www.pivotaltracker.com/story/show/{162190563})

#### Screenshots (if appropriate)
<img width="1440" alt="screen shot 2018-12-04 at 4 54 14 am" src="https://user-images.githubusercontent.com/9039613/49418034-b63b1500-f780-11e8-8e91-d61855f5fab6.png">
#### Questions:
